### PR TITLE
fix url

### PR DESCRIPTION
**Pull request by LinshanYu**

### CODE REVIEW

File: communication/meeting-notes-archive/201902-201911_Community_Meeting_Notes.md [hunk 1]
```diff
@@ -1126,7 +1126,7 @@ Twitter | [Sep 19th](https://twitter.com/stephenaugustus/status/1174797710043430
 *   [ 0:00 ] **📣Announcements 📣**
     *   More testgrid stuff I forgot if we have time (prefer shoutouts if we don’t) [Aaron Crickenberger]
         *   [We moved a lot of existing job configs over to annotations for you](https://github.com/kubernetes/test-infra/pull/13205)
-        *   [Testgrid changes (annotations or not) must pass tests](https://github.com/kubernetes/test-infra/blob/master/testgrid/cmd/configurator/config_test.go)
+        *   [Testgrid changes (annotations or not) must pass tests](https://github.com/kubernetes/test-infra/blob/31b2a35/testgrid/cmd/configurator/main_test.go)
```
Comment (idvoretskyi) [inline communication/meeting-notes-archive/201902-201911_Community_Meeting_Notes.md]: Why not the `master` branch?

Comment (idealhack) [reply communication/meeting-notes-archive/201902-201911_Community_Meeting_Notes.md]: see https://github.com/kubernetes/community/pull/4318#discussion_r358099759

**Pull request merged**